contracts/multihop/src/tests/query.rs

```diff
@@ -299,22 +299,23 @@ fn simulate_swap_single_pool_with_fees() {
 
     // 1000 tokens initially
     // swap 300 from token1 to token2 with 2000 bps (20%)
-    // tokens2 will be 240
-    assert_eq!(simulated_swap.ask_amount, 240i128);
+    // the lp has a maximum allowed bps of 1_500 (15%)
+    // tokens2 will be 255
+    assert_eq!(simulated_swap.ask_amount, 255i128);
     assert_eq!(
         simulated_swap.commission_amounts,
-        vec![&env, (String::from_str(&env, "FZY"), 60i128)]
+        vec![&env, (String::from_str(&env, "FZY"), 45i128)]
```

Comment:
Why would those outputs changed now?

Reply:
the way it's set now is we can have `maximum_allowed_bps` of %15. 

In this test in particular we initialize the admin initialized the pool with %20, but because that is over the cap (%15) we automatically initialize the pool with the cap limit - %15. 

One thing we haven't talked about - how much the cap should be - I put %15 entirely by my own whim. Also, whenever the admin tries to create a pool that violates that cap, should we throw an error, instead of reducing the `maximum_allowed_bps`?

Reply:
```rust
        // if the swap_fee_bps is above the threshold, we set it to the threshold value
        if swap_fee_bps > MAXIMUM_ALLOWED_TOTAL_FEE_BPS {
            swap_fee_bps = MAXIMUM_ALLOWED_TOTAL_FEE_BPS;
        }
```
Then this is wrong, if the fee is too big just error and cancel. Why would we want to hide that something is wrong?